.github/workflows/ci-pr-validation.yml

```diff
@@ -167,6 +167,14 @@ jobs:
           docker run -i -v $PWD:/pulsar-client-node build:latest \
               /pulsar-client-node/pkg/linux/build-napi-inside-docker.sh
 
+      - name: Test NAPI file in other containers
+        if: matrix.image == 'linux_glibc'
+        run: |
+          ./tests/load-test.sh node:16-buster
```

Comment:
Can we keep the param node version consistent with  `${{matrix.nodejs}}`?

Reply:
Why? The `load-test.sh` is used to test the `Pulsar.node` works for various versions of Node.js. Did you mean testing Node.js 18 instead of 19?

Reply:
> The load-test.sh is used to test the Pulsar.node works for various versions of Node.js.

I think that we should test on various types of operating systems(such as centos7, etc). But both use Node.js 18.

Reply:
The `node:<version>-<suffix>` image just represents a specific node version and OS. For example, `node:16-buster` represents Node.js 16 and Debian-buster.

CentOS 7 has already been verified when building the `Pulsar.node`. There are many Linux distributions and they cannot be covered all. And it will require some extra efforts to install Node.js on them.

Reply:
Just as the GLIBCXX compatibility issue here https://github.com/apache/pulsar-client-node/pull/288#discussion_r1073622557, installing Node.js 18 on some systems could be very complicated. The only way is to compiling it from source. Since NAPI guarantees the ABI compatibility, why do we insist on verifying with Node.js 18?

Reply:
These tests are added to verify `Pulsar.node` can be loaded by Node.js with higher version libstdc++.so dependency.